instrumentation/opentelemetry-instrumentation-wsgi/src/opentelemetry/instrumentation/wsgi/__init__.py

```diff
@@ -436,15 +452,10 @@ def __call__(self, environ, start_response):
             start_response: The WSGI start_response callable.
         """
         req_attrs = collect_request_attributes(environ)
-        active_requests_count_attrs = {}
-        for attr_key in _active_requests_count_attrs:
-            if req_attrs.get(attr_key) is not None:
-                active_requests_count_attrs[attr_key] = req_attrs[attr_key]
-
-        duration_attrs = {}
-        for attr_key in _duration_attrs:
-            if req_attrs.get(attr_key) is not None:
-                duration_attrs[attr_key] = req_attrs[attr_key]
+        active_requests_count_attrs = parse_active_request_count_attrs(
```

Comment:
this should be replaced with otel_wsgi.parse_active_request_count_attrs()

Reply:
No, this is function exists in the same file

Reply:
ok got it